Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -14,20 +14,33 @@
 
 import Foundation
 
+#if os(Linux)
+  import Glibc
+#else
+  import Darwin
+#endif
```

Comment:
Do we still need those imports here?

Reply:
Yeah, they're required for the `fflush` call. :-(

Reply:
Huh... glad I checked. Apparently they're not... at least not with the toolchain I'm using. Let me throw it at CI with these removed.